hw/ip/lc_ctrl/data/lc_ctrl.hjson

```diff
@@ -5,57 +5,459 @@
   clock_primary: "clk_i",
   bus_device: "tlul",
   bus_host: "none",
+  regwidth: "32",
 
-//  ///////////////////////////
-//  // Interrupts and Alerts //
-//  ///////////////////////////
-//
-//  interrupt_list: [
-//  ],
-//
-//  alert_list: [
-//  ],
-//
-//  ////////////////
-//  // Parameters //
-//  ////////////////
-//  param_list: [
-//  ]
-//
-//  /////////////////////////////
-//  // Intermodule Connections //
-//  /////////////////////////////
-//  // TODO: these need to be refined during implementation and integration
-//  inter_signal_list: [
-//  ] // inter_signal_list
-
-  ///////////////
-  // Registers //
-  ///////////////
 
-  regwidth: "32",
+  ///////////////////////////
+  // Interrupts and Alerts //
+  ///////////////////////////
+
+  // This module has no interrupts
+  no_auto_intr_regs: "True"
```

Comment:
o do we need to add this? i thought just by not having an interrupt_list would be enough.

Reply:
yeah that is needed, otherwise reggen allocates empty registers for the IRQ... maybe we should derive this automatically.

Reply:
yeah feels like that might be a good idea.

Reply:
ok, I've fixed that. we can now just omit the IRQ lists.